webui/react/src/pages/TrialDetails/TrialDetailsLogs.tsx

```diff
@@ -171,7 +171,6 @@ const TrialDetailsLogs: React.FC<Props> = ({ experiment, trial }: Props) => {
 
     return () => {
       fieldCanceler.abort();
-      canceler.current.abort();
```

Comment:
This works good, but `canceler` wont abort if we remove it, right?
What do you think to change like this?
```ts
      canceler.current.abort();
      const newCanceler = new AbortController();
      canceler.current = newCanceler;
```

Reply:
I agree